### ISSUE TITLE
Upload chart release to Rekor

### ISSUE THREAD

**Issue by sabre1041** [sigstore/helm-charts#5]
Publish Chart releases to Rekor using the [Helm Sigstore plugin](https://github.com/sigstore/helm-sigstore)

**Comment by sabre1041** [comment]
Dependent on #4

**Comment by hectorj2f** [comment]
I like this one.

**Comment by hectorj2f** [comment]
@sabre1041 is anyone working on this plugin? I would like to help out.

**Comment by sabre1041** [comment]
@hectorj2f an initial implementation is [here](https://github.com/sigstore/helm-sigstore/pull/3) if you want to test it out. It will be merged in the next day or so

**Comment by developer-guy** [comment]
Hello folks, this is a really really great feature! we've started another issue on cosign project side, maybe kinda bit related because both are Helm plugin features, we are waiting for feedbacks, please feel free to tell us your thoughts, all would be appreciated 🤩🤝

👉 https://github.com/sigstore/cosign/issues/1118

**Comment by sabre1041** [comment]
Resolved in #48
